README.md

```diff
@@ -1,207 +1,47 @@
-# Template for setting up reproducible projects
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors-)
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
+# Open Seeds Self-organised 
 
-*This is a template for setting up a research or data science project online, such as on GitHub.*
+Welcome! This is the [Self-Organised](https://we-are-ols.org/openseeds/) (OLS- Self-Organised) program repository. 
+This repository will be used to provide resources for individuals and communities who want to run the OLS Open Seeds program but couldn't get a chance to participate during the earlier cohorts. Also, the resources here could be useful to folks who want to run the program self-space and independently
```

Comment:
```suggestion
Here, we provides tools, practices and resources for individuals and communities who want to organise the Open Seeds program for their respective communities using existing training materials, documentation and most importantly, the rich collection of pre-recorded videos from OLS's Open Seeds Video Library.
This can also be used for people to follow Open Seeds training in self-paced manner independently.
```